src/components/modal/veiledergruppe/veileder-gruppe-modal.tsx

```diff
@@ -89,6 +90,7 @@ export function VeilederGruppeModal(props: VeilederModalProps) {
     }
 
     function slettVeiledergruppeOgLukkModaler() {
+        logEvent('portefolje.metrikker.veiledergrupper.slettknapp');
```

Comment:
hva hvis slettingen feiler?

Reply:
sånn jeg forstod det skal vi bare logge når veilederne trykker på slett-knappen eller avbryt-knappen her, også vil det også logges dersom toasten for sletting-velykket vises, og hvis feilmodalen for sletting vises, men det er igjen to andre metrikker

Reply:
https://jira.adeo.no/browse/FO-2622

9. Antall trykk på "slett"-knappen i slett-modal 
10. Antall trykk på “avbryt”-knappen i slett-modal 